tests/onnx/test_onnx_v2.py

```diff
@@ -284,6 +284,13 @@ def _onnx_export(self, test_name, name, model_name, feature, onnx_config_class_c
         model_class = FeaturesManager.get_model_class_for_feature(feature)
         config = AutoConfig.from_pretrained(model_name)
         model = model_class.from_config(config)
+
+        # It seems `nn.AdaptiveAvgPool2d` with `output_size` > 1 is not working with ONNX. We get `Segmentation fault`
+        # without any useful information.
+        if model.__class__.__name__ == "Data2VecVisionForSemanticSegmentation":
```

Comment:
Maybe we should just skip this test for `Data2VecVisionForSemanticSegmentation`?